tests/test_deploy_github_pages.py

```diff
@@ -63,9 +61,9 @@ def test_pushing_to_existing_docu_branch_same_source(setup_test_env):
     commit_id_new = current_commit_id.stdout
 
     # the docu files where updated, so a new commit should be pushed to the remote
-    assert not commit_id_new == ""
-    assert not commit_id_old == ""
-    assert not commit_id_old == commit_id_new
+    assert not commit_id_new == "" \
+           and not commit_id_old == "" \
+           and not commit_id_old == commit_id_new
```

Comment:
Is there a particular reason to update it this way? @Nicoretti 

By the way, PEP8 E127 warning occurs due to indentations. That's why I removed the indentations.

Reply:
That's fine, I screwed up the indentation, so :+1: . 

This are some of the reasons why I personally prefer the `()` syntax over the `\` syntax:
* It is easier to maintain the `()` version. Lines just can be added or removed without worrying if it will break the expression (otherwise you always need to keep in mind to at `\` at the right place).
* If you use the `\` syntax your focus gets split `1. the actual code`, `2.  the correctness of the multi line syntax`. ->> less noise and therefore reads cleaner/easier << -